iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -289,49 +280,33 @@ class ConversationListViewController: ViewController,
             do {
                 try await self.conversationListController.loadNextConversations(limit: .channelsPageSize)
             } catch {
-                logDebug(error)
+                logError(error)
             }
             self.isLoadingConversations = false
         }.add(to: self.taskPool)
     }
+}
 
-    // MARK: - UIScrollViewDelegate
```

Comment:
All of the scrolling logic about centering on a cell or updating the centered cell is in the layout class.